cross/expat/PLIST

```diff
@@ -1,4 +1,5 @@
 bin:bin/xmlwf
 lnk:lib/libexpat.so
 lnk:lib/libexpat.so.1
-lib:lib/libexpat.so.1.6.9
+lib:lib/libexpat.so.1.7.0
+rsc:lib/pkgconfig/expat.pc
```

Comment:
PLIST must contain files to install only.
pkgconfig files are used at build time only and must not be included here.